docs/CLUSTERCTL.md

```diff
@@ -1,62 +1,85 @@
-# Clusterctl 
+# Clusterctl
 
 <a name="version_interop"></a>
+
 ## Version Interop
+
 The compatible versions of Core CAPI / clusterctl and CAPVCD are listed in the table below:
 
 | CAPVCD Version | Core CAPI/clusterctl Version |
-|----------------|------------------------------|
-| main           | v1.4.0                       |
+| -------------- | ---------------------------- |
+| main           | v1.5.4                       |
+| 1.3.z          | v1.5.4                       |
+| 1.2.z          | v1.4.0                       |
 | 1.1.z          | v1.4.0                       |
 | 1.0.z          | v1.1.3                       |
 
 <a name="clusterctl_set_up"></a>
-## Set up
+
+## Set up (Applicable only for 1.1.z and older)
```

Comment:
Richard, I see that you updated the section title to "Applicable only for 1.1.z and older", but I suggest removing the section completely for all the branches >= 1.2.z

We want to avoid ifs and buts in the documentation - zero confusion for the readers

Reply:
I suggest adding a point here to refer to the relevant branch documentation to get an accurate list of steps. It is hard to capture everything in the main branch.